File: ABMA/SwiftUI/Data/Entities/Sponsor+CoreDataProperties.m
//
//  Sponsor+CoreDataProperties.m
//  ABMA
//
//  Created by Nate Condell on 1/15/24.
//  Copyright © 2024 Nathan Condell. All rights reserved.
//
//

#import "Sponsor+CoreDataProperties.h"

@implementation Sponsor (CoreDataProperties)

+ (NSFetchRequest<Sponsor *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"Sponsor"];
}

@dynamic bObjectId;
@dynamic created;
@dynamic imageUrl;
@dynamic updated;
@dynamic url;
@dynamic year;

@end

File: ABMA/Data/Entities/Sponsor+CoreDataProperties.swift
//
//  Sponsor+CoreDataProperties.swift
//  ABMA
//
//  Created by Nate Condell on 2/11/24.
//  Copyright © 2024 Nathan Condell. All rights reserved.
//
//

import Foundation
import CoreData


extension Sponsor {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Sponsor> {
        return NSFetchRequest<Sponsor>(entityName: "Sponsor")
    }

    @NSManaged public var bObjectId: String?
    @NSManaged public var created: Date?
    @NSManaged public var imageUrl: String?
    @NSManaged public var updatedAt: Date?
    @NSManaged public var url: String?
    @NSManaged public var year: Year?

}

extension Sponsor : Identifiable {

}
